src/OrchardCore.Modules/OrchardCore.Users/Views/Items/RemoveUserRoleTask.Fields.Edit.cshtml

```diff
@@ -0,0 +1,16 @@
+@using OrchardCore.Users.Workflows.ViewModels
+@model RemoveUserRoleTaskViewModel
+
+<div class="mb-3" asp-validation-class-for="UserName">
+    <label asp-for="UserName">@T["UserName"]</label>
+    <input type="text" asp-for="UserName" class="form-control code" />
+    <span asp-validation-for="UserName"></span>
+    <span class="hint">@T["The User to update. With Liquid support."]</span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="RoleName">
+    <label asp-for="RoleName">@T["RoleName"]</label>
+    <input type="text" asp-for="RoleName" class="form-control code" />
```

Comment:
Why text box instead of checkbox list? The view model can be checkbox and the activity can be a string expression

Reply:
I don't think you need liquid. But you can add a toggler that switch between liquid and checkbox. I think check box list is must easier from a user perspective. I would say in most "if not all" cases you are selected a predefined role name and no need to use Liquid.